SPECS/initscripts/initscripts.spec

```diff
@@ -27,187 +24,128 @@ Provides:       network-scripts = %{version}-%{release}
 This package contains the script that activates and deactivates most
 network interfaces, some utilities, and other legacy files.
 
-%package -n debugmode
```

Comment:
Just curious, why are we removing the -debugmode subpackage?

Reply:
It no longer gets built in versions >= 9.78
https://src.fedoraproject.org/rpms/initscripts/blob/rawhide/f/initscripts.spec#_636